htdocs/product/reassort.php

```diff
@@ -131,7 +131,7 @@
 $sql = 'SELECT p.rowid, p.ref, p.label, p.barcode, p.price, p.price_ttc, p.price_base_type, p.entity,';
 $sql .= ' p.fk_product_type, p.tms as datem,';
 $sql .= ' p.duration, p.tosell as statut, p.tobuy, p.seuil_stock_alerte, p.desiredstock,';
-$sql .= ' SUM(s.reel) as stock_physique';
+$sql .= ' SUM(distinct s.reel) as stock_physique';
```

Comment:
If there is two lines with stock 2 (2 in warehouse A and 2 in warehouse B), you will get 2 instead of 4

You should have no problem of duplicate because of link with category because there is a test 
AND cp.fk_categorie = id category
So record link with product id and category id is always unique.
May be your trouble is the one fixed by this commit 15dca77330e3412e7fb445a20f3aecbf32c84eac ?

Reply:
Hi Eldy,

I just installed a clean copy of Dolibarr 14.0.5, created two categories, one product and one warehouse.
I assigned 10 units to the product for this warehouse.
I assigned both categories to this product.
When I click on the search symbol in /product/reassort.php shows 20 units, exactly the double amount.
So this is indeed a bug which has to be solved.

![image](https://user-images.githubusercontent.com/3024019/152530141-6decd506-3081-4984-a57b-ff69374535f5.png)^
![image](https://user-images.githubusercontent.com/3024019/152530469-9d5cf2e4-19c2-424e-8246-d16642ffdd35.png)
![image](https://user-images.githubusercontent.com/3024019/152530624-991df720-b4af-490d-8efe-05ef288c24f0.png)

Reply:
Can you paste the sql you have in the list ?  Did you apply the fix of https://github.com/Dolibarr/dolibarr/commit/15dca77330e3412e7fb445a20f3aecbf32c84eac ?

Reply:
Thank you. The problem was solved with your patch.